module_checksum_upgrade/README.rst

```diff
@@ -68,6 +68,8 @@ Contributors
 * Juan José Scarafía <jjs@adhoc.com.ar>
 * Jairo Llopis <jairo.llopis@tecnativa.com>
 
+The main idea of this module comes from `module_auto_update` by LasLabs Inc.
```

Comment:
Please reword to avoid confusing readers: you can credit LasLabs for their [initial implementation](https://github.com/OCA/server-tools/pull/882), but [the idea comes from somebody else](https://github.com/OCA/server-tools/issues/790#issuecomment-298837926). :wink:

Reply:
I did not know the idea was yours. Credits updated.